t/Makefile.am

```diff
@@ -458,7 +458,6 @@ check_LTLIBRARIES = \
 	shell/plugins/conftest.la \
```

Comment:
Commit message for ` testsuite: avoid shell standalone mode in initrc tests`
- `flux_shell_setop(3)` missing `t`
- closing paren and period are reversed